src/Util/Configuration.php

```diff
@@ -346,7 +346,7 @@ public function getListenerConfiguration(): array
                     continue;
                 }
 
-                if (!$node->tagName === 'arguments') {
```

Comment:
PHPStan reported:
`Strict comparison using === between bool and string will always evaluate to false.`